src/java/org/apache/cassandra/io/util/File.java

```diff
@@ -128,12 +128,25 @@ public File(Path path)
 
     /**
      * Try to delete the file, returning true iff it was deleted by us. Does not ordinarily throw exceptions.
+     * @deprecated use {@link File#deleteIfExists()}, see alsoe {@link PathUtils#tryDelete(Path)}
```

Comment:
```suggestion
     * @deprecated use {@link File#deleteIfExists()}, see also {@link PathUtils#tryDelete(Path)}
```

Reply:
Is there a reason to not just remove this function rather than deprecate it?

Reply:
It would require many changes to the existing code (refactoring). But at least marking it as deprecated gives it a better visibility now, so if anyone does another ticket where this function is called, they would see it highlighted by the IDE and maybe check if the usage is correct.

Reply:
I don’t like marking some API as deprecated if we are leaving it in use internally. All you are doing is adding warning spam to the build.

Reply:
I undeprecated those methods, as there is simply too much code calling into them, and analyzing why we're deleting files and ignoring some errors in each of those places is out of scope of this work.